lib/generator.ts

```diff
@@ -542,22 +570,83 @@ export class Generator extends Cli {
     return source
   }
 
+  private sourceLines(source: string) {
```

Comment:
Suggest using some verb as function name for consistency reasons:
- getSourceLines
- splitSourceToLines
- ...